### ISSUE TITLE
PHP 7.1 build fails due Simple PHPUnit

### ISSUE THREAD

**Issue by aik099** [minkphp/Mink#782]
I'm not sure why, but on PHP 7.1 The `vendor/bin/simple-phpunit` script (part of https://symfony.com/doc/current/components/phpunit_bridge.html) is choosing to use PHPUnit 5.7 version.

Such decision results in a failure, when attempting to download chosen PHPUnit version, because on GitHub there is no PHPUnit 5.7 version, but there is PHPUnit 5.7.0 version.

Not sure if this is bug in Simple PHPUnit or out `.travis.yml` file.

Failing build URL: https://travis-ci.org/minkphp/Mink/jobs/555679617

Failing build Output:

```
PHP Warning:  fopen(https://github.com/sebastianbergmann/phpunit/archive/5.7.zip): failed to open stream: HTTP request failed! HTTP/1.0 404 Not Found
 in /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit on line 41
PHP Stack trace:
PHP   1. {main}() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:0
PHP   2. fopen() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:41
Warning: fopen(https://github.com/sebastianbergmann/phpunit/archive/5.7.zip): failed to open stream: HTTP request failed! HTTP/1.0 404 Not Found
 in /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit on line 41
Call Stack:
    0.0004     416008   1. {main}() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:0
    0.0035     416952   2. fopen() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:41
PHP Warning:  stream_copy_to_stream() expects parameter 1 to be resource, boolean given in /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit on line 41
PHP Stack trace:
PHP   1. {main}() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:0
PHP   2. stream_copy_to_stream() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:41
Warning: stream_copy_to_stream() expects parameter 1 to be resource, boolean given in /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit on line 41
Call Stack:
    0.0004     416008   1. {main}() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:0
    0.3663     422776   2. stream_copy_to_stream() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:41
PHP Warning:  chdir(): No such file or directory (errno 2) in /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit on line 53
PHP Stack trace:
PHP   1. {main}() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:0
PHP   2. chdir() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:53
Warning: chdir(): No such file or directory (errno 2) in /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit on line 53
Call Stack:
    0.0004     416008   1. {main}() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:0
    0.3666     422416   2. chdir() /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge/bin/simple-phpunit:53
No composer.json in current directory, do you want to use the one at /home/travis/build/minkphp/Mink/vendor/symfony/phpunit-bridge? [Y,n]? 
```

**Comment by stof** [comment]
this happens in the `prefer-lowest` job. We need to increase the lowest bound of the phpunit-bridge constraint as old releases have been broken by the removal of branches in the PHPUnit github repo.

**Comment by aik099** [comment]
Then this is bug in PHPUnit Bridge itself as I've suspected.

We can specify `SYMFONY_PHPUNIT_VERSION` env variable manually and then specified version of PHPUnit will be used by PHPUnit Bridge.

**Comment by stof** [comment]
The issue is not about the version of PHPUnit. Older versions of the bridge were downloading the github archive of the branch of the PHPUnit repo. And PHPUnit deleted all unmaintained branches. Newer versions of the bridge are downloading the latest tag of each minor version instead. So bumping the min version of the bridge will do it.

**Comment by aik099** [comment]
It seems to be resolved by #783, but I'm not sure.
